### DISCUSSION TITLE
Specify only one channel

### DISCUSSION THREAD

**Discussion by ale94mleon** [fcakyon/conda-publish-action#8] [Q&A] [answered]
Hi, I am using this action and so far so good. However, a new conda-recipe that I am creating has really nasty dependencies and only works if `conda-forge` is the only channel. How could be specified only this channel in the action?
Thanks in advance!

**Comment by fcakyon** [comment, marked as answer]
You may have to create a fork, update this line and use that as your action :+1: https://github.com/fcakyon/conda-publish-action/blob/02852ad7af3e3ba5e1f1083a2914ad4479f26af3/entrypoint.sh#L28
